doc/audit/merkle_tree.rst

```diff
@@ -1,14 +1,30 @@
 Merkle Tree
 ===========
 
-The high-integrity guarantees of CCF are enforced by a single :term:`Merkle Tree` which records all transactions that mutate the key-value store. The root of the Merkle Tree is regularly signed by the primary node and the signature is recorded in the ``ccf.signatures`` key-value map. Like any other transaction, the signature transaction is also recorded in the ledger, which allows for offline auditability of the service.
-
 .. note:: The :ref:`CCF Python package <quickstart/install:Python Package>` provides the ``ledger`` module to easily parse the ledger. More details :doc:`here <python_library>`.
 
-The signature transaction also contains the Merkle Tree so far (note: the tree is compacted at regular interval to avoid infinite memory growth).
+The high-integrity guarantees of CCF are enforced by a single :term:`Merkle Tree` which records the cryptographic hash (leaves) of all transactions that mutate the key-value store. The root of the Merkle Tree is regularly signed by the primary node and the signature is recorded in the ``public:ccf.internal.signatures`` key-value map. Like any other transaction, the signature transaction is also recorded in the ledger, which allows for offline auditability of the service.
+
+The following diagram demonstrates how the integrity of the ledger can be verified: a signature transaction (at ``seqno`` 6) signs the root of the Merkle Tree so far (after ``seqno`` 5).
+
+.. image:: ../img/merkle_single_signature.svg
+  :width: 600
```

Comment:
On my screen, the images seem a little small compared to the text, maybe just make them the same width as the text?

Reply:
Same here. I've updated the images so that there isn't that much white space around them, which makes them look bigger.